src/test/config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "serverURL": "wss://carta.asiaa.sinica.edu.tw/socketdev",
+    "serverURL": "wss://carta.asiaa.sinica.edu.tw/socketicd",
```

Comment:
Better to be /socketdev

Reply:
Thanks!!! I forget to change back to the socketdev!!!